### ISSUE TITLE
обновить Download

### ISSUE THREAD

**Issue by SemyonKuznetsov** [jdi-testing/jdn-ai#896]

**Comment by SemyonKuznetsov** [comment]
иногда скачивается java файл, иногда архив, как это работает? что должно скачиваться? надо ли это описывать в readme?

**Comment by AlexeyGirin** [comment]
> иногда скачивается java файл, иногда архив, как это работает? что должно скачиваться?
> 
> вот такие архивы ![image](https://user-images.githubusercontent.com/101407186/211798399-9a144762-d947-4ee9-9254-c81ccae3f4a6.png)

Давайте поменяем тултипы - если пользователь скачивает всё (через кнопку) то скачивается весь проект со всеми ПО
Если пользователь скачивает конкретный ПО (через меню ПО) - то скачивается только жава файл с этим ПО

**Comment by SemyonKuznetsov** [comment]
@MariiaNebesnova надо посмотреть есть ли нужное в апи и будет ли это вообще работать

**Comment by MariiaNebesnova** [comment]
yes, it's possible. we need to use https://github.com/GoogleChromeLabs/browser-fs-access
